catalog/acm/Kptfile

```diff
@@ -0,0 +1,12 @@
+apiVersion: kpt.dev/v1
+kind: Kptfile
+metadata:
+  name: gke-acm
```

Comment:
Should we make this a blueprint under catalog/gke (Similar to Terraform where its a submodule)

Reply:
Yes, there is a plan to compose this in a GKE blueprint. That would be the next step after this standalone blueprint is done.